data/linux-bridge/003-bridge-marker-rbac.yaml

```diff
@@ -1,4 +1,3 @@
-apiVersion: extensions/v1beta1
```

Comment:
Is that not necessary anymore?

Reply:
it's a duplication, see 2 lines below there is already `apiVersion: rbac.authorization.k8s.io/v1beta1`